.sisyphus/plans/2026-04-30-share-link-api/plan.md

```
@@ -0,0 +1,157 @@
+  ]
+}
+```
+
+message_range 필터링:
+- from_message_id / to_message_id 둘 다 null → 전체 messages
+- non-null → `WHERE message_id >= $from AND message_id <= $to`
+
+검증: is_deleted=false AND (expires_at IS NULL OR expires_at > now())
+
+**DELETE /api/v1/chats/{thread_id}/share**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)